### ISSUE TITLE
Need demo that opens/closes dialog, and accessibility checklist

### ISSUE THREAD

**Issue by ghost** [PolymerElements/paper-dialog-scrollable#5]
If this is a demo of a dialog, we need the demo to show off the typical interaction: have a button that pops up the dialog, ideally not full-screen.

Things to check:
- Should have role=dialog, aria-modal=true/false appropriately
- Opening the dialog should place focus inside, or if there's nothing focusable, on the dialog element itself (tabindex=-1)
- When the dialog is open, other controls behind it on the page should be inert and all content outside the dialog should be aria-hidden
- Esc should close the dialog
- If the dialog has a title, use aria-labelledby to reference the title from the dialog element if possible. If impossible due to shadow boundaries, extract the title text and set the dialog's aria-label to that text.

**Comment by morethanreal** [comment]
@tjsavage Would you mind moving this to PolymerElements/paper-dialog-behavior?

**Comment by tjsavage** [comment]
auto-moving this issue to PolymerElements/paper-dialog-behavior#3 and closing this one
